runtime/moonbase/src/lib.rs

```diff
@@ -1202,16 +1202,6 @@ impl OnRuntimeUpgrade for MaintenanceHooks {
 	fn try_on_runtime_upgrade(checks: bool) -> Result<Weight, TryRuntimeError> {
 		AllPalletsWithSystem::try_on_runtime_upgrade(checks)
 	}
-
-	#[cfg(feature = "try-runtime")]
-	fn pre_upgrade() -> Result<Vec<u8>, sp_runtime::DispatchError> {
-		AllPalletsWithSystem::pre_upgrade()
-	}
-
-	#[cfg(feature = "try-runtime")]
-	fn post_upgrade(state: Vec<u8>) -> Result<(), sp_runtime::DispatchError> {
-		AllPalletsWithSystem::post_upgrade(state)
-	}
```

Comment:
What are these changes for? Was it something we missed in the dependency upgrade (and probably broke `try-runtime`)?

Reply:
After the recent changes in `polkadot-sdk` and our changes in `moonkit`, these overrides are redundant. Their existence is neither harmful nor helpful.